UPEI-Android/group-project-mindjogg
# Dev moodpicker

### Summary of PR 
Adds Mood trackers full feature


### Source
Closes 
#238 
#217 
#216
#182
#181 


### Review and Testing Steps

1.  Pull this branch
2. Start server and app (change IP)
3. Login and navigate to Mood Tracker screen
4. Pick a mood write a note 
5. Save the mood
6.  Go to history screen 
7.  You will see the graph and history of your picked mood

### Additional Information (If applicable)
Check if animation is working properly for graph

### Note 
For now you can delete mood once it is save (Future feature)
